.github/workflows/main.yml

```diff
@@ -34,13 +34,9 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python }}
-    - name: Install tox
-      run: |
-        python -m pip install --upgrade pip
-        pip install tox
     - name: Test
       run: |
-        tox -e ${{ matrix.tox_env }}  
+        pipx run tox -e ${{ matrix.tox_env }}  
```

Comment:
This is news to me, so GitHub actions come with `pipx` pre-installed?

Reply:
yes, and the pipx tox is not broken wrt some details